netbox/dcim/api/views.py

```diff
@@ -672,6 +673,12 @@ def list(self, request):
         peer_device_name = request.query_params.get(self._device_param.name)
         peer_interface_name = request.query_params.get(self._interface_param.name)
 
+        # check permissions of user, if login is required
+        if settings.LOGIN_REQUIRED:
+            if not request.user.has_perm('dcim.view_device') or not request.user.has_perm('dcim.view_interface'):
```

Comment:
Instead of calling `has_perm()` twice, we can pass a list of both permissions to `request.user.has_perms()`.

Reply:
OK, thanks.